src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/LrdApiController.java

```diff
@@ -17,66 +17,77 @@
+        }
     )
     @ApiResponses({
-            @ApiResponse(
-                    code = 200,
-                    message = "Successfully retrieved list of Service Code or Ccd Case Type Details",
-                    response = LrdOrgInfoServiceResponse.class,
-                    responseContainer = "list"
-            ),
-            @ApiResponse(
-                    code = 400,
-                    message = "Bad Request"
-            ),
-            @ApiResponse(
-                    code = 401,
-                    message = "Forbidden Error: Access denied"
-            ),
-            @ApiResponse(
-                    code = 404,
-                    message = "No Service found with the given ID"
-            ),
-            @ApiResponse(
-                    code = 500,
-                    message = "Internal Server Error"
-            )
+        @ApiResponse(
+            code = 200,
+            message = "Successfully retrieved list of Service Code or Ccd Case Type Details",
+            response = LrdOrgInfoServiceResponse.class,
+            responseContainer = "list"
+        ),
+        @ApiResponse(
+            code = 400,
+            message = "Bad Request"
+        ),
+        @ApiResponse(
+            code = 401,
+            message = "Forbidden Error: Access denied"
+        ),
+        @ApiResponse(
+            code = 404,
+            message = "No Service found with the given ID"
+        ),
+        @ApiResponse(
+            code = 500,
+            message = "Internal Server Error"
+        )
     })
     @GetMapping(produces = APPLICATION_JSON_VALUE)
     public ResponseEntity<Object> retrieveOrgServiceDetails(
         @RequestParam(value = "serviceCode", required = false) String serviceCode,
         @RequestParam(value = "ccdCaseType", required = false) String ccdCaseType,
         @RequestParam(value = "ccdServiceNames", required = false) String ccdServiceNames) {
         log.info("Inside retrieveOrgServiceDetails");
-        List<LrdOrgInfoServiceResponse> lrdOrgInfoServiceResponse = null;
-
         long requestParamSize = Stream.of(serviceCode, ccdCaseType, ccdServiceNames)
             .filter(StringUtils::isNotBlank)
             .count();
         if (requestParamSize > 1) {
             throw new InvalidRequestException("Please provide only 1 of 3 params: "
                                                   + "serviceCode, ccdCaseType, ccdServiceNames.");
         }
+        String inputValue = Stream.of(serviceCode, ccdCaseType, ccdServiceNames)
+            .filter(StringUtils::isNotBlank).collect(Collectors.joining());
+        inputValue = StringUtils.trim(inputValue);
+        if (Pattern.compile(REG_EXP_WHITE_SPACE).matcher(inputValue).matches()
+            || !Pattern.compile(REG_EXP_SPCL_CHAR).matcher(inputValue).matches()) {
+            return ResponseEntity.status(400).body("Bad request, message contains special characters or white spaces");
```

Comment:
rather than return please throw exception change error message 
throw new InvalidRequestException("Param contains special characters. ',' comma and '_' underscore allowed only");